src/transformers/models/auto/configuration_auto.py

```diff
@@ -318,7 +318,6 @@
         ("transfo-xl", "TRANSFO_XL_PRETRAINED_CONFIG_ARCHIVE_MAP"),
         ("unispeech", "UNISPEECH_PRETRAINED_CONFIG_ARCHIVE_MAP"),
         ("unispeech-sat", "UNISPEECH_SAT_PRETRAINED_CONFIG_ARCHIVE_MAP"),
-        ("upernet", "UPERNET_PRETRAINED_CONFIG_ARCHIVE_MAP"),
```

Comment:
There is no `UPERNET_PRETRAINED_CONFIG_ARCHIVE_MAP`. Let me know if you want to add it in `configuration_upernet.py` and keep this line untouched here, @NielsRogge

Reply:
Tbh I've never understood what UPERNET_PRETRAINED_CONFIG_ARCHIVE_MAP is used for 😅

Reply:
The only place I find is 
https://github.com/huggingface/transformers/blob/1b37fb5e17add5e65f7076f4e88225e11156c107/src/transformers/commands/add_new_model_like.py#L690
which uses `CONFIG_ARCHIVE_MAP_MAPPING_NAMES` to get model names. Missing some entry won't fail the method though.

@sgugger I guess you will say it's not worth adding `upernet` to `CONFIG_ARCHIVE_MAP_MAPPING_NAMES`, considering the low usage (for now) and the fact of we don't want to maintain such archive maps in the future, rigth?

Reply:
If it's not added then `ALL_PRETRAINED_CONFIG_ARCHIVE_MAP` will be wrong (in the sense it won't have all architectures) but I don't know if we care about that.